src/game/all_combinations.go

```diff
@@ -0,0 +1,26 @@
+package game
+
+
+// allCombinations := []string{"100000100000100000100000100000000000","000000100000100000100000100000100000","010000010000010000010000010000000000","000000010000010000010000010000010000","001000001000001000001000001000000000","000000001000001000001000001000001000","000100000100000100000100000100000000","000000000100000100000100000100000100","000010000010000010000010000010000000","000000000010000010000010000010000010","000001000001000001000001000001000000","000000000001000001000001000001000001","111110000000000000000000000000000000","011111000000000000000000000000000000","000000111110000000000000000000000000","000000011111000000000000000000000000","000000000000111110000000000000000000","000000000000011111000000000000000000","000000000000000000111110000000000000","000000000000000000011111000000000000","000000000000000000000000111110000000","000000000000000000000000011111000000","000000000000000000000000000000111110","000000000000000000000000000000011111","100000010000001000000100000010000000","000000010000001000000100000010000001","000000100000010000001000000100000010","010000001000000100000010000001000000","000001000010000100001000010000000000","000000000010000100001000010000100000","000010000100001000010000100000000000","000000000001000010000100001000010000"}
+
+var ALL_COMBINATIONS_OF_WIN = [32]int64{34905131008, 545392672, 17452565504, 272696336, 8726282752, 136348168, 4363141376, 68174084, 2181570688, 34087042, 1090785344, 17043521, 66571993088, 33285996544, 1040187392, 520093696, 16252928, 8126464, 253952, 126976, 3968, 1984, 62, 31, 34630287488, 270549121, 541098242, 17315143744, 1108378624, 34636832, 2216757248, 17318416}
```

Comment:
Maybe using [integer literals](https://golang.org/ref/spec#Integer_literals) will avoid you the duplication.

Reply:
I didn't understand where I have duplication ? The first commented line was for debugging purpose and should be removed.

Reply:
well, using integer literals you could actually "see" the grids. Especially since you're allowed to put a non-significant underscore in an integer literal to ease readability. 

E.g. `&b001000_001000_001000_001000_001000_000000` is more expressive than `8726282752`

Reply:
Clear, thank you :+1: